app/themes/user-admin/src/app/user-event-log/user-event-log.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, OnInit } from '@angular/core';
+
+import { UsersService }   from '../shared/users.service';
+import { Event } from '../shared/models/event.model';
+
+import { UserListComponent } from '../user-list/user-list.component';
+
+import { Subscription } from "rxjs";
+
+@Component({
+  selector: 'app-user-event-log',
+  templateUrl: './user-event-log.component.html',
+  styleUrls: ['./user-event-log.component.scss'],
+  providers: [UsersService]
+})
+export class UserEventLogComponent implements OnInit {
+
+  //Variable for the loading animation.
+  private wait: boolean;
+
+  //initialize the subscription
+  private events$: Subscription;
+  private events: Event[];
+
+  //Parameters of getUserEvents()
+  private userID = '1';
```

Comment:
This ID should not be hardcoded, the User ID should come from the router itself (like a query string value).

Check out these docs for more info:
- https://angular.io/docs/ts/latest/guide/router.html#!#route-parameters
- http://stackoverflow.com/a/40276279/6357846

If you need an working example, take a look at this the `ngOnInit` method in the user-edit component itself: https://github.com/Erdiko/user-admin/blob/master/app/themes/user-admin/src/app/user-edit/user-edit.component.ts#L48